feat(stack): add defaults for align prop

ninamarina opened the pull request:
## Description
https://dialpad.atlassian.net/browse/DLT-1265
Adds defaults for aligning items in the Stack component.
Documentation will be updated in the `dialtone-vue` repository.

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [x] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [ ] Update, remove, or extend all affected documentation.
 - [x] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).

## Obligatory GIF (super important!)
![](path/to/gif)

juliodialpad reviewed (changes requested):
I think we don't need to have those as utility classes.

I'd be better to just set those default values for row and column variants of the stack component.

And just say in the documentation that by default row stack will have `align-items: center` and column stack has `justify-content: flex-start` set by default.

ninamarina commented:
> I think we don't need to have those as utility classes.
> 
> I'd be better to just set those default values for row and column variants of the stack component.
> 
> And just say in the documentation that by default row stack will have `align-items: center` and column stack has `justify-content: flex-start` set by default.

Changed in 

> I think we don't need to have those as utility classes.
> 
> I'd be better to just set those default values for row and column variants of the stack component.
> 
> And just say in the documentation that by default row stack will have `align-items: center` and column stack has `justify-content: flex-start` set by default.

Changed in 3353c31b30dff7b8467815fd54323fee3038d21a

iropolo commented on lib/build/less/components/stack.less:
```diff
@@ -17,18 +17,30 @@
   &--column {
     --stack-direction: column;
+
+    align-items: normal;
```
`normal;` is the default value of `align-items` Is this adding value here?

iropolo replied:
Would remove it if it is not adding value. 
in the other hand, `justify-content` is adding something necessary to work correctly.

ninamarina replied:
I take your suggestion, let's have the less lines of css we can 👍

iropolo reviewed (approved):
pre-approving it.

Have some doubt on the use of `align-items: normal` and `justify-content: normal` but if it is intended to solve some issue or edge case would be cool to review it here, if it is not I would remove them.

Thanks,

iropolo commented on lib/build/less/components/stack.less:
```diff
@@ -17,18 +17,30 @@
     --stack-direction: row;
+
+    align-items: center;
+    justify-content: normal;
```
duh, right, didnt see it is under `&--row {` 👍🏼

braddialpad reviewed (approved):
Looks good!

juliodialpad reviewed (approved):
Looking good, thanks Nina!

I'll create a ticket to improve this component, instead of using media queries for responsive versions we should be using our dialtone generators function but that's another story. I'll create the ticket and refer back to this PR.

juliodialpad commented:
Cleanup stack component ticket: https://dialpad.atlassian.net/browse/DLT-1270
